torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -116,6 +117,10 @@ def check_node_safe(node: Node):
     """
     SAFE_TORCH_MODULES = ("torch.functional", "torch.nn.functional")
 
+    SAFE_HOPS = {
+        wrap.tag_activation_checkpoint,
+    }
```

Comment:
Two things I want feedback on (from @jamesjwu)
1) is this safe to cache? (what's the definition of cacheable)
2) we have a separate is_cacheable flag for the fx_graph_cache. Do we want to just reuse that right now, or are we OK with having separate allow/deny lists for aotautograd cache vs fx_graph_cache. 
https://github.com/pytorch/pytorch/blob/86d7d39bffd3b7b099310fb351b2b36f99981d6f/torch/_higher_order_ops/wrap.py#L149